test/colony-task-work-rating.js

```diff
@@ -155,29 +169,34 @@ contract("Colony Task Work Rating", () => {
 
       await forwardTime(SECONDS_PER_DAY * 5 + 1, this);
       await colony.revealTaskWorkRating(1, WORKER_ROLE, WORKER_RATING, RATING_2_SALT, { from: EVALUATOR });
+
       const roleWorker = await colony.getTaskRole.call(taskId, WORKER_ROLE);
-      assert.isTrue(roleWorker[1]);
```

Comment:
This could be changed to `assert.isFalse` too rather than just deleted. I'm giving the ones further down a pass, because we're checking extra things instead, but here we're just making the test less strict.

Reply:
I deleted it because `rated` is no longer part of the flow and IMO shouldn't be tested in tests outside of the ones which test for backwards compatibility; since this test is only concerned with the evaluator's evaluation (and doesn't deal with the worker's evaluation, whether submitted on time submitted late), there's no value of the `rateFail` boolean that we could meaningfully check.

I'm happy to add a test for this boolean in the backwards-compatibility test that you wrote.